docs/configuration.md

```diff
@@ -381,7 +381,7 @@ section of the configuration.
   * It uses configuration from `exchange.pair_whitelist` and `exchange.pair_blacklist`.
 * `VolumePairList`
   * Formerly available as `--dynamic-whitelist [<number_assets>]`. This command line
-option is deprecated and should no longer be used.
+option was removed and can no longer be used.
```

Comment:
i would just remove it from the documentation - no point in mentioning that we removed it (it'll only confuse new users)

Reply:
Agree, removed.